.github/workflows/cpp-packaging.yml

```diff
@@ -561,8 +561,8 @@ jobs:
               # MacOS: use LLVM binutils for both X64 and ARM64
               additional_flags+=(-L)
             elif [[ "${{ matrix.sdk_platform }}" == "windows" && "${variant}" == *"/x64/"* ]]; then
-              # Windows x64: force input and output target format
-              additional_flags+=(-f pe-x86-64,pe-bigobj-x86-64)
+              # Windows x64: force input and output target format and use LLVM binutils for correct objcopy (issue #793)
+              additional_flags+=(-f pe-x86-64,pe-bigobj-x86-64 -L)
```

Comment:
I think you need to remove `-f pe-x86-64,pe-bigobj-x86-64` because `llvm-objcopy` don't support triples written this way, and it works without that anyway.